src/json.cpp

```diff
@@ -1241,7 +1241,6 @@ bool JsonIn::get_bool()
     }
     err << "not a boolean value!  expected 't' or 'f' but got '" << ch << "'";
     error( err.str(), -1 );
```

Comment:
IIRC one of the static checker will yell at you about this.
```suggestion
    error( err.str(), -1 );
    throw JsonError( "warnings are silly" );
```

Reply:
Nevermind, I think I was thinking of somewhere else.